wagtailio/project_styleguide/templates/patterns/_pattern_library_only/streamfields.html

```diff
@@ -7,4 +7,5 @@
 {% include "patterns/components/streamfields/highlight_block/highlight_block.html" with image_on_right=True %}
 {% include "patterns/components/streamfields/logo_block/logo_block.html" %}
 {% include "patterns/components/streamfields/quotes/quotes.html" %}
-{% include "patterns/components/streamfields/cards/cards_list_block.html" %}
\ No newline at end of file
+{% include "patterns/components/streamfields/cards/cards_list_block.html" %}
+{% include "patterns/components/streamfields/sign-up-form/sign-up-form.html" %}
```

Comment:
```suggestion
{% include "patterns/components/sign_up_form/sign_up_form.html" %}
```
Suggestion: move this into the `streamfields` dir